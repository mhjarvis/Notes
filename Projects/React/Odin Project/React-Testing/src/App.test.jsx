// testing setup via:
// https://www.robinwieruch.de/vitest-react-testing-library/

import { describe, it, expect } from "vitest";

describe("something truthy and falsy", () => {
    it("true to be true", () => {
        expect(true).toBe(true);
    });
    it("false to be false", () => {
        expect(false).toBe(false);
    });
});
