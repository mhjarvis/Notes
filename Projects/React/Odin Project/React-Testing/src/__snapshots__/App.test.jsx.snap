// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`App component > renders magnificent monkeys 1`] = `
<div>
  <div
    style="display: flex; flex-direction: column; align-items: center; justify-content: center; text-align: center; width: 100vw;"
  >
    <button>
      Click Me
    </button>
    <h1>
      Magnificent Monkeys
    </h1>
  </div>
</div>
`;
